InformationScripting/test/scripts/countChanges.py

```diff
@@ -0,0 +1,17 @@
+# script $<"<changes>" | "<countChanges>" | "<heatmap>">$
```

Comment:
What does it matter if it is implemented in C++ or in python? What you suggest is what I previously had, i.e. this specific use case. Now we have a more generic output and can use it in many ways. I'm not sure what the advantages would be if such cases would be supported by C++ implementation.